operator/test/test.sh

```diff
@@ -144,6 +144,13 @@ test_metrics() {
 }
 
 test_chains() {
+  echo -n "  - Check Deployment: "
```

Comment:
per clarification/confirmation in stand up this check needs to move to the checks in install.sh around the https://github.com/openshift-pipelines/pipeline-service/blob/main/operator/images/cluster-setup/content/bin/install.sh#L137-L143 area

Reply:
In that case it is already there: 
https://github.com/openshift-pipelines/pipeline-service/blob/8619a4e55ac4ffeee4d3fc6d54712aeaf73075e3/operator/images/cluster-setup/content/bin/install.sh#L138
this line checks for all deployment status

Reply:
This is what I wanted to point to but couldn't find. I have removed the checks from test.sh